packages/toolpad-utils/src/fs.ts

```diff
@@ -3,8 +3,22 @@ import * as path from 'path';
 import { Dirent } from 'fs';
 import * as yaml from 'yaml';
 import { yamlOverwrite } from 'yaml-diff-patch';
+import * as prettier from 'prettier';
 import { errorFrom } from './errors';
 
+/**
+ * Formats a yaml source with `prettier`.
+ */
+const PRETTIERNAME = 'prettier.config.js';
+
+export function formatYaml(code: string): string {
+  const readConfig = prettier.resolveConfig.sync(PRETTIERNAME);
+  return prettier.format(code, {
+    parser: 'yaml',
```

Comment:
You may move `parser: 'yaml',` underneath `...readConfig,` to indicate we intend it to be an override and not a default value.